packages/vuetify/src/styles/components/_selection-controls.sass

```diff
@@ -76,6 +76,7 @@
     left: -12px
     top: calc(50% - 24px)
     margin: 7px
+    pointer-events: none // Enable title attr hover effect, as ripple blocks the input
```

Comment:
Have you tested this in all of our supported browsers?

Reply:
Actually now that you mentioned it, I've noticed that it causes a new bug of ripple not disappearing (probably on `mouseout`); but because we've already moved the title to the parent `v-input`, the ripple already inherits the title hover effect, so this workaround isn't necessary anymore, and I've removed it in a new commit.